library/proc_macro/src/lib.rs

```diff
@@ -1333,7 +1379,47 @@ impl Literal {
     // was 'c' or whether it was '\u{63}'.
     #[unstable(feature = "proc_macro_span", issue = "54725")]
     pub fn subspan<R: RangeBounds<usize>>(&self, range: R) -> Option<Span> {
-        self.0.subspan(range.start_bound().cloned(), range.end_bound().cloned()).map(Span)
+        bridge::client::FreeFunctions::literal_subspan(
```

Comment:
Shouldn't this be a "static method" on `Span`?

Reply:
I can move it to be a normal method on `Span` as it actually uses no other parts of `Literal` in its implementation currently, If we need to add in extra parameters for validation in the future, we can add those back in when needed.